skuba-update/skuba_update/skuba_update.py

```diff
@@ -256,10 +249,10 @@ def is_zypper_error(code):
     return code != 0 and code < ZYPPER_EXIT_INF_UPDATE_NEEDED
 
 
-def is_restart_needed(code):
+def zypper_needs_transaction_restart(code):
```

Comment:
what is the _transaction_ good for? this is just an update stack update (not really a transaction restart afaik). I would find something like 

```
zypper_packagemanager_needs_restart()
```

would be more intuitive to me (thats at least matching the nomenclature in the man page)

Reply:
Yup, even better name! Thanks!